.gitignore

```diff
@@ -154,3 +154,4 @@ grcov.tar.bz2
 lcov.info
 *.profraw
 lcov.info
+config/.secrets.toml
```

Comment:
[`.env``](https://github.com/forgeflux-org/interface/blob/146a2cd7f5afc93fdd9ff3affc26b8770a412f1b/config/.env-sample) file serve the same purpose of `.secrets.toml`. It's the alternative to passing secrets via env vars